west.yml

```diff
@@ -168,7 +168,7 @@ manifest:
       groups:
         - hal
     - name: hal_intel
-      revision: 89a0db5d9155484624447d62a5211bcf6f6f0387
+      revision: 7826c074c05d419e712dc03694ddae06bb4c583f
```

Comment:
Is this change in revision intentional? If so this should likely be done in another commit

Reply:
yes, intentional. HAL git has recent patch for better support of I2C SEDI shim-layer driver.
without this change, build of I2C SEDI driver won't pass.

Reply:
Yes so please make the module update a commit describing just that

Reply:
separate west.yml change into a dedicated commit with comments.